kubeflow-namespace/base/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
I think you should include cert-manager as well like
https://github.com/kubeflow/gcp-blueprints/blob/master/kubeflow/instance/kustomize/namespaces/namespaces.yaml

I might suggest the following hierarchy

namespaces/all/kustomization.yaml

This way if we need to create different combinations of namespaces we can do so.

Reply:
Actually, you might need to keep "base" for legacy reasons as kfctl depends on it today.

Reply:
sure, will change like that

Reply:
@jlewi PTAL